src/Stuart/Converters/JsonToSchedulingSlots.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Stuart\Converters;
+
+class JsonToSchedulingSlots
+{
+    public static function convert($json)
+    {
+        $body = json_decode($json);
+        $zone = new \Stuart\Zone();
```

Comment:
I am not sure the client library should bring this level of complexity, the information provided by the zone are not very useful for integrations.